Automattic/jetmon
# Migrate to Node 18

This PR upgrades the Docker env to use Node 18.

### To test

1. Apply PR
2. Rebuild Docker env
3. Run Jetmon
4. Make sure it works.
